### PR TITLE
Fixed template information (version and extensions fields)

### LINKED ISSUE #54: [Bug]: Wording on Question dialog needs to be updated - Github issue
### Contact Details

susandou@amazon.com

### What happened?

On the Question template, there is a section titled 'What's the question?', immediately followed by the sentence 'Tell us about your doubts'.  That sentence should be replaced (though doubting and questioning are sometimes linked in dictionaries and thesaurus, their meanings aren't identical).

I would suggest replacing it with something like:

Provide detailed information about your question below.

### Version

BABEL_1_X_DEV (Default)

### Extension

babelfishpg_tsql (Default)

### What flavour of Linux are you running into the bug?

Other

### Relevant log output

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by 3manuek**
### Description

Fixed incorrect references, sentences, and requested information in the issue templates.

- Removed contact details request.
- Rephrasing of values and details to match styling guidelines.
 
### Issues Resolved

No issues related.

### Check List
- [x] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is under the terms of the Apache 2.0 and PostgreSQL licenses, and grant any person obtaining a copy of the contribution permission to relicense all or a portion of my contribution to the PostgreSQL License solely to contribute all or a portion of my contribution to the PostgreSQL open source project.

For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/babelfish-for-postgresql/babelfish_extensions/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/enhancement.yaml [hunk 1]
```diff
@@ -34,16 +34,15 @@ body:
     validations:
       required: false 
   - type: dropdown
-    id: version
+    id: extension
     attributes:
-      label: Version
-      description: What version of our software are you running?
+      label: Extension
+      description: What extension contrib of our software are you running?
```
Comment (nasbyj) [inline .github/ISSUE_TEMPLATE/enhancement.yaml]: We should actually remove this from the enhancement template; we should decide what extension a feature request affects.

File: .github/ISSUE_TEMPLATE/question.yaml [hunk 2]
```diff
@@ -26,6 +26,27 @@ body:
       value: "I have a question!"
     validations:
       required: true
+  - type: dropdown
```
Comment (nasbyj) [inline .github/ISSUE_TEMPLATE/question.yaml:19]: Also not relevant for questions.

Comment (nasbyj) [reply .github/ISSUE_TEMPLATE/question.yaml:19]: Please remove this chunk as well.

**Review by nasbyj — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/question.yaml (hunk 2 above)
Comment (nasbyj) [reply .github/ISSUE_TEMPLATE/question.yaml:19]: Nevermind, version here makes sense.

**Review by nasbyj — DISMISSED** [review]

**Review by gopinathpai — DISMISSED** [review]
looks good to me

File: .github/ISSUE_TEMPLATE/bug.yaml [hunk 3]
```diff
@@ -2,35 +2,33 @@ name: Babelfish for PostgreSQL Extensions Bug Report
...
       required: false
   - type: textarea
     id: what-happened
     attributes:
       label: What happened?
-      description: Also tell us, what did you expect to happen?
-      placeholder: Tell us what you see!
+      description: Please provide details about the bug, and the steps needed to recreate the bug.
+      placeholder: Tell us about the bug.
       value: "A bug happened!"
```
Comment (susanmdouglas-aws) [inline .github/ISSUE_TEMPLATE/bug.yaml]: I think we discussed removing the 'value' content, but if I misunderstood, can we replace 'A bug happened!' with:

Tell us about the bug.

I'd rather have that line repeated twice than sound too excited about a bug...

File: .github/ISSUE_TEMPLATE/bug.yaml [hunk 4]
```diff
@@ -39,19 +37,18 @@ body:
     id: extension
     attributes:
       label: Extension
-      description: What extension is the problem occurring with?
+      description: If known, what extension is the problem occurring with?
```
Comment (nasbyj) [inline .github/ISSUE_TEMPLATE/bug.yaml]: s/what/which/

**Review by susanmdouglas-aws — APPROVED** [review]

**Comment by 3manuek** [comment]
@nasbyj @susanmdouglas-aws I reopened this PR into a new one from a personal fork as requested: https://github.com/babelfish-for-postgresql/babelfish_extensions/pull/205. We can probably close this one.

**Pull request closed without merging**